cli/pcluster/config_sanity.py

```diff
@@ -530,6 +530,56 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
         # FSX FS Id check
         elif resource_type in ["fsx_fs_id", "FSx_storage_capacity", "FSx_imported_file_chunk_size", "FSx_export_path"]:
             self.__validate_fsx_parameters(resource_type, resource_value)
+        elif resource_type == "EFA":
+            valid_instances = get_supported_efa_instances(self.region)
```

Comment:
shall we move this to a __validate_efa_parameters function? Ideally we should also start writing unit tests for the new code we add to pcluster cli.